examples/monitors-clock-tasks/1/mark_unknown.json

```diff
@@ -0,0 +1,6 @@
+{
+  "type": "mark_unknown",
+  "ts": 1714072962,
+  "monitor_environment_id": 1,
+  "checkin_id": 1
+}
```

Comment:
I thought that when we detected an outage, we'd just set a variable somewhere so that when we go to mark a miss, we mark an unknown instead?

Reply:
Yeah we will. This is for marking all in progress as unknown status